AmazonWeb1/src/main/java/com/amazon/dao/impl/DataSourceDBConnection.java

```diff
@@ -0,0 +1,42 @@
+package com.amazon.dao.impl;
+
+import com.amazon.exception.DBException;
+import org.apache.commons.dbcp.BasicDataSource;
+
+import javax.sql.DataSource;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+
+public class DataSourceDBConnection {
+
+    private static final DataSourceDBConnection DATA_SOURCE_DB_CONNECTION = new DataSourceDBConnection();
```

Comment:
What is the use of DataSourceConnection?

Reply:
this class provides the method for getting a database connection.